python/packages/autogen-ext/src/autogen_ext/models/anthropic/config/__init__.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, Dict, List, Literal, Optional, Union
+
+from autogen_core.models import ModelCapabilities, ModelInfo  # type: ignore
+from pydantic import BaseModel
+from typing_extensions import TypedDict
+
+
+class ResponseFormat(TypedDict):
+    type: Literal["text", "json_object"]
```

Comment:
It can also be a BaseModel for structured output

Reply:
I cant seem to see any documentation on support for structured output. 
https://github.com/anthropics/anthropic-cookbook/blob/main/misc/how_to_enable_json_mode.ipynb

Reply:
Ok